tests/security/apparmor/aa_autodep.pm

```diff
@@ -49,6 +50,14 @@ sub run {
             \}/sxx
     };
 
+    # Check the test files exist, double check to avoid perfomance issue
+    my $ret = script_run "ls -1 $test_binfiles > tee /dev/$serialdev";
+    if ($ret) {
```

Comment:
Lili,

Performance purpose to use this ?

Reply:
> Lili,
> 
> Performance purpose to use this ?

Good question. 
When this test case failed randomly, e.g., http://openqa.nue.suse.com/tests/5094046#step/aa_autodep/18
I am not sure if the original "$test_binfiles" do not exist atm or the "aa_autodep" did not generate the expected profiles atm, so I added 2 workarounds, this workaround is a kind of performance purpose or a debugging purpose.